docs/logging/home-page.md

```diff
@@ -0,0 +1,52 @@
+---
+id: tofu.logging.home title: Tofu Logging
+---
+
+# Welcome to the Tofu Logging Home page
+
+This is a set of tools for logging. Configurable, concise, handy.
+
+## Installation
+
+Add these dependencies into your project:
+
+```sbt
+libraryDependencies += "tf.tofu" %% "tofu-logging" % "<latest version in the badge in README>"
+libraryDependencies += "tf.tofu" %% "tofu-logging-derivation" % "<latest version in the badge in README>"
+
+```
+
+For ZIO users the following is enough:
+
+```sbt
+libraryDependencies += "tf.tofu" %% "tofu-logging-zio" % "<latest version in the badge in README>"
+```
+
+## How to use
+
+The usage of logging is described as a set of recipes, suitable for many cases.
+Check it out [here](recipes/recipes.md).
+
+## Quick demo
+
+```scala
+@derive(loggable)
```

Comment:
I think imports need to be added since most people (according to my humble opinion) will not go further and just use derivation and syntax